pkg/cloud/services/compute/servergroup.go

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
we'll want unit tests for this one.

Reply:
Done, moved them into separate file, but they were in `instance_test.go`.